src/racer/primitive.rs

```diff
@@ -78,6 +78,7 @@ impl PrimKind {
             LitIntType::Unsuffixed => PrimKind::U32,
         }
     }
+    #[allow(dead_code)]
```

Comment:
Please do **nothing** for these 2 functions.
They're left for integer method completion but I still haven't decided to implement it for 2.2 or not.